processing/src/main/java/org/apache/carbondata/processing/merger/CompactionType.java

```diff
@@ -26,7 +26,6 @@
 public enum CompactionType {
     MINOR,
     MAJOR,
-    IUD_UPDDEL_DELTA,
```

Comment:
Don't remove ENUM, it may give compatibility issues.

Reply:
I have modified code according to your suggestion